examples/double_spend.rs

```diff
@@ -0,0 +1,367 @@
+#[cfg(all(test, not(all(feature = "server", feature = "client"))))]
+compile_error!("tests need both server and client features");
+
+use std::{
+    io,
+    io::{BufRead, BufReader},
+    iter::{repeat_with, Extend},
+    net::{Ipv4Addr, SocketAddr},
+    sync::{
+        atomic::{AtomicU16, Ordering},
+        Arc,
+    },
+    time::{Duration, Instant},
+};
+
+use duct::cmd;
+use futures::future::join_all;
+//use futures::executor::block_on;
+use tokio::{net::TcpStream, task::yield_now};
+use tracing::info;
+use tracing_subscriber;
+use structopt::StructOpt;
+use rand::{thread_rng, Rng};
+
+const CLIENT_BIN: &str = "/home/mhadji/research/at2/at2-node/target/debug/client";
+const SERVER_BIN: &str = "/home/mhadji/research/at2/at2-node/target/debug/server";
```

Comment:
Sorry for the absolute paths, `CARGO_BIN_EXE_<name>` seems to be not exposed for the rust examples

Reply:
Indeed, [it's "only set when building an integration test or benchmark"](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates). You can however use another variable, `CARGO_MANIFEST_DIR`, which is the root of the crate.

```suggestion
const CLIENT_BIN: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/target/debug/client");
const SERVER_BIN: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/target/debug/server");
```

Reply:
Ah nice! Thank you :)